src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/VariantOfMapFW.java

```diff
@@ -0,0 +1,304 @@
+            return map16RO;
+        case THREE:
+            return map8RO;
+        default:
+            throw new IllegalStateException("Unrecognized kind: " + kind());
+        }
+    }
+
+    @Override
+    public MapFW<KV, VV> getAs(
+        EnumWithInt8 kind,
+        int kindPadding)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public VariantOfMapFW<KV, VV> tryWrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        if (super.tryWrap(buffer, offset, maxLimit) == null)
+        {
+            return null;
+        }
+        EnumWithInt8FW enumWithInt8 = enumWithInt8RO.tryWrap(buffer, offset, maxLimit);
+        if (enumWithInt8 == null)
+        {
+            return null;
+        }
+        switch (kind())
+        {
+        case ONE:
+            if (map32RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        case TWO:
+            if (map16RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        case THREE:
+            if (map8RO.tryWrap(buffer, offset + enumWithInt8.sizeof(), maxLimit) == null)
+            {
+                return null;
+            }
+            break;
+        default:
+            break;
+        }
+        if (limit() > maxLimit)
+        {
+            return null;
+        }
+        return this;
+    }
+
+    @Override
+    public VariantOfMapFW<KV, VV> wrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        super.wrap(buffer, offset, maxLimit);
+        EnumWithInt8FW enumWithInt8 = enumWithInt8RO.wrap(buffer, offset, maxLimit);
+        switch (kind())
+        {
+        case ONE:
+            map32RO.wrap(buffer, offset + enumWithInt8.sizeof(), maxLimit);
+            break;
+        case TWO:
+            map16RO.wrap(buffer, offset + enumWithInt8.sizeof(), maxLimit);
+            break;
+        case THREE:
+            map8RO.wrap(buffer, offset + enumWithInt8.sizeof(), maxLimit);
+            break;
+        default:
+            break;
+        }
+        checkLimit(limit(), maxLimit);
+        return this;
+    }
+
+    @Override
+    public VariantOfMapFW<KV, VV> wrapWithKindPadding(
+        DirectBuffer buffer,
+        int elementsOffset,
+        int maxLimit,
+        int kindPadding)
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public String toString()
+    {
+        return get().toString();
+    }
+
+    @Override
+    public int limit()
+    {
+        return get().limit();
+    }
+
+    public static final class Builder<KB extends Flyweight.Builder<KV>, KV extends Flyweight, VB extends Flyweight.Builder<VV>,
+        VV extends Flyweight> extends VariantOfFW.Builder<VariantOfMapFW<KV, VV>, EnumWithInt8, MapFW>
```

Comment:
Suggest `KV` -> `K`, `VV` -> `V`, and changing type order to align with `MapFW` feedback.